people/iva.json

```diff
@@ -5,6 +5,7 @@
   "email": "iva.zajacova@mild.blue",
   "title_cs": "Head of Operations",
   "title_en": "Head of Operations",
+  "title_de": "Einsatzleiter",
```

Comment:
hele to ne, to je neco jako velitel... to se spis hodi pro poradkovou cetu :)

a obecne tedy německé překlady zatím dost pokulhávají... nechal bych zatím head of operations

Reply:
Ok, ponechavam Head of Operations.